src/components/ImageAlignmentPage.vue

```diff
@@ -332,9 +361,22 @@
 
      cancel() {
        this.$router.go(-1);
+     },
+
+     toggleHints() {
+       let x = document.getElementById("hints");
```

Comment:
Oh no, you should use `v-show` or `v-if` checking a new `showHints` variable added to `data()`.